tests/ha/iscsi_client.pm

```diff
@@ -61,12 +61,12 @@ sub run {
     for (1 .. 40) { send_key 'backspace'; }
     type_string 'iqn.1996-04.de.suse:01:' . get_hostname . '.' . get_cluster_name;
     wait_still_screen 3;
-    send_key 'alt-v';    # discoVered targets
+    is_sle("=15-SP6") ? send_key 'alt-d' : send_key 'alt-v';    # discovered targets
```

Comment:
Minor comment, I would suggest you add `leap15.6` here, as well

Reply:
Not sure if the module is tested in leap15.6. Let me check that. Thanks.